Client/Frontend/Library/BookmarkDetailPanel.swift

```diff
@@ -358,6 +358,7 @@ class BookmarkDetailPanel: SiteTableViewController {
                 }
 
                 cell.indentationLevel = min(item.indent, maxIndentationLevel)
+                cell.separatorInset = UIEdgeInsets(top: 0, left: CGFloat(cell.indentationLevel) * cell.indentationWidth + 61, bottom: 0, right: 0)
```

Comment:
Same type of comment here, could we have a constant with a relevant name for this 61? So it's easier to understand context when we come back to this later on.

Reply:
61 is the current separator inset for bookmark cells without any indentation. Although it seems that only Safari indents the separator for nested bookmark folders so if it's probably a design decision.